pkg/astnormalization/enum_type_extending.go

```diff
@@ -13,12 +13,22 @@ func extendEnumTypeDefinition(walker *astvisitor.Walker) {
 	walker.RegisterEnterEnumTypeExtensionVisitor(&visitor)
 }
 
+func extendEnumTypeDefinitionKeepingOrphans(walker *astvisitor.Walker) {
```

Comment:
instead of heaving a specialized constructor it would be better to have a constructor with optional options.

```go
func extendEnumTypeDefinition(walker *astvisitor.Walker, opts ...Option) {
```

See: https://github.com/TykTechnologies/graphql-go-tools/blob/master/pkg/astnormalization/astnormalization.go#L110

It will help to keep the code extensible